Cargo.toml

```diff
@@ -51,8 +52,10 @@ serde_cbor = "0.11"
 hex = { version = "0.4", features = ["serde"] }
 
 [features]
+default = []
 std = []
-virtual = ["std", "vpicc"]
+virtual = ["std", "vpicc", "virt"]
+virt = ["std", "trussed/virt"]
```

Comment:
Maybe rename `virtual` to `vpicc`?

Reply:
Let's not add things to this PR, which is already large enough (even more true for #127 ). I'll make an issue and another PR. I'll also rename it in PIV.